nolearn/lasagne/base.py

```diff
@@ -83,6 +83,17 @@ def __getstate__(self):
         return state
 
 
+def objective(layers,
+              loss_function,
+              target,
+              aggregate=aggregate,
+              **kwargs):
+    output_layer = list(layers.values())[-1]
```

Comment:
`output_layer = list(layers.values())[-1]` appears in at least two places. Perhaps this should be a utility function of some sort?